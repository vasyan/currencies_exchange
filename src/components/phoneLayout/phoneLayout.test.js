import React from 'react'
import { shallow } from 'enzyme'
import { shallowToJson } from 'enzyme-to-json'
import PhoneLayout from './index'

describe('<PhoneLayout>', () => {
  it('should render', () => {
    expect(shallowToJson(shallow(<PhoneLayout />))).toMatchSnapshot()
  })
})
