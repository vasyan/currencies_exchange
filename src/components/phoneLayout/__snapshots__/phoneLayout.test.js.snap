// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<PhoneLayout> should render 1`] = `
<div
  className="phone"
>
  <div
    className="phone_shade"
  />
  <div
    className="phone_speaker"
  />
  <div
    className="phone_screen"
  />
  <div
    className="phone_button"
  />
</div>
`;
